app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -64,6 +64,26 @@ class DradisDatatable {
       },
       lengthChange: false,
       pageLength: 25,
+      stateSave: true,
+      stateSaveCallback: function(settings, data) {
+        var dataTableDomId = settings.oInstance[0].id;
```

Comment:
If this is the same as `var dataTableDomId = this.$table.attr('id');`, then I think we can save this as a class variable since it will never change.

Reply:
Opted to use `data-local-storage-key="key"` instead as it provides more flexibility